src/rove_slam/config/ekf.yaml

```diff
@@ -24,3 +24,48 @@ ekf_filter_node:
                       false, false, false,
                       false, false, false,
                       false, false, false]
+
+ekf_filter_node_global:
+    ros__parameters:
+        frequency: 30.0
+        two_d_mode: false
+        publish_acceleration: true
+        publish_tf: true
+
+        map_frame: map              
+        odom_frame: odom            
+        base_link_frame: base_footprint
+        world_frame: map           
+
+        odom0: /odom
+        odom0_config: [true,  true,  true,
+                       false, false, true,
+                       false, false, false,
+                       false, false, true,
+                       false, false, false]
+                       
+        imu0: /imu
+        imu0_config: [false, false, false,
+                      true,  true,  true,
+                      false, false, false,
+                      false, false, false,
+                      false, false, false]
+
+        odom1: /odometry/gps
+        odom1_config: [true,  true,  false,
+                       false, false, false,
+                       false, false, false,
+                       false, false, false,
+                       false, false, false]
+
+navsat_transform:
```

Comment:
I don't think this should be there (gps config and not ekf). But arguable

Reply:
both need to be used with each other, but can be seperated